docs/instructions.md

```diff
@@ -221,46 +220,45 @@ PREDICT_MODULE=finance_income_prediction.predict
 PREDICT_FUNCTION=predict_adult_income_rf
 ```
 
-1.  ROUTES: defines endpoint to create and function to call when endpoint is invoked
+1. `ROUTES`: defines the endpoint to create and function to call when endpoint is invoked as follows:
 `serviceroute:predict-module-name:predict-function-name`
-2. TRAIN_MODULE: defines python module in which train function is defined
-3. TRAIN_FUNCTION: train function name
-4.  PREDICT MODULE AND PREDICT FUNCTION: same as train but for predict.
-
-> in case of using daemon s2i builder we only need to specify ROUTES.
-PREDICT_MODULE && PREDICT FUNCTION are optional when using `s2i` daemon builder image. Though we  provide it in all our examples for readability
+2. `TRAIN_MODULE`: defines the python module in which train function is defined
+3. `TRAIN_FUNCTION`: the train function name
+4. `PREDICT MODULE` AND `PREDICT FUNCTION`: same as for train, but for the predict function
 
-here:
+NOTE: In the case of using the daemon s2i builder you only need to specify `ROUTES`.
+`PREDICT_MODULE` and `PREDICT FUNCTION` are optional when using the `s2i` daemon builder image, though it is used in all examples for readability.
 
-`adult_income_rf/predict` is the web-service route
-`finance_income_prediction.predict` is the python module in which predict functionality is defined
-`predict_adult_income_rf` is the predict function name defined in the `finance_income_prediction.predict` module shown above
+**Example:**
 
+`adult_income_rf/predict` is the web-service route.
+`finance_income_prediction.predict` is the python module in which predict functionality is defined.
+`predict_adult_income_rf` is the predict function name defined in the `finance_income_prediction.predict` module.
 
-## all
 
-All models can be trained and packaged together as well, exposing predict for an entire project/usecase(with 3 models here) as one single container with multiple routes for each of those models.
+## All
 
-> note `all` creates a new predict container with all routes in exactly the same way as individual models are created. Hence it needs the same files, some of which like .s2i/environment can be auto generated
+Models can be trained and packaged together, exposing predict for an entire project or use case as one single container with multiple routes for each of those models.(exemplified with the three models here)
```

Comment:
not entirely obvious which the three models are. I'm assuming you're referring to the example for finance income prediction.